packages/backend-core/src/middleware/downloadBody.ts

```diff
@@ -0,0 +1,22 @@
+import { Ctx } from "@budibase/types"
+
+/**
+ * Expects a standard "query" query string property which is the JSON body
+ * of the request, which has to be sent via query string due to the requirement
+ * of making an endpoint a GET request e.g. downloading a file stream.
```

Comment:
There's no downloading happening here which is a bit misleading for the name of the middleware. Could we rename it some more clear, `queryToBody`? Might also want to add the restriction that this only affects `GET` requests, as there is the potential to misuse and overwrite the body of a post request.